app/src/main/res/values/themes.xml

```diff
@@ -52,6 +52,22 @@
     <item name="android:textColorPrimary">@color/component_color_shared_primary_dark_text_color</item>
   </style>
 
+  <style name="AlertDialogBackgroundPositiveButtonTheme" parent="Theme.MaterialComponents.Light.Dialog.Alert.Bridge">
+    <item name="android:paddingBottom">3dp</item>
+    <item name="android:color">@color/component_color_shared_dialogs_radio_button_unchecked_color</item>
+    <item name="buttonBarPositiveButtonStyle">@style/AlertDialogPositiveButtonBackgroundTheme</item>
+  </style>
+
+  <style name="AlertDialogPositiveButtonBackgroundTheme">
+    <item name="android:background">@drawable/positive_button_rounded_border_solid_color_primary</item>
+    <item name="android:textColor">@color/color_palette_white_text_color</item>
```

Comment:
I believe that for dark mode support we need to be using component_colors. See [wiki](https://github.com/oppia/oppia-android/wiki/Dark-Mode).